### ISSUE TITLE
Can't add new keyboard due to Missing '"' character (OS_X)

### ISSUE THREAD

**Issue by BenDover178** [michaeldickens/Typing#22]
I'm trying to add a new keyboard to the program (31 keys; adding the apostrophe) but for some reason it won't register. When I try to open tools.c in terminal I get the error messages in the photo. 
<img width="935" alt="adding keyboard errors" src="https://user-images.githubusercontent.com/38433419/45652765-8128fb80-ba8a-11e8-81a6-13ed4a68f5ed.png">
It's trying to say there's no ending " character even though there is (and it doesn't look any different compared to the full standard board). Any ideas of what this might be?

**Comment by michaeldickens** [comment]
Hi Ben,

Looks like that's because you put an unescaped quotation mark in the middle
of the string. You need to put a backslash in front of the quotation mark,
so instead of

"qwertyuiopasdfghjkl;'zxcvbnm,./QWERTYUIOPASDFGHJKL:"ZXCVBNM<>?"

it should be

"qwertyuiopasdfghjkl;'zxcvbnm,./QWERTYUIOPASDFGHJKL:\"ZXCVBNM<>?"

Hope this fixes your problem.

**Comment by BenDover178** [comment]
Yes thanks. I guess I just glossed over it for the standard board as an input character.

I tried running the algorithm for a few seconds and it's formatting the layouts like so:
 Y  P  O  U  /   K  D  L  C  W
 I  N  E  A  ;   M  H  T  S  R
 V  Q  <  >  ?   Z  B  F  G  J
 X 
 y  p  o  u  :   k  d  l  c  w
 i  n  e  a  ,   m  h  t  s  r
 v  q  "  '  .   z  b  f  g  j
 x 

Is the key on the bottom supposed to be the apostrophe key?
ex:
 Y  P  O  U  /   K  D  L  C  W
 I  N  E  A  ;   M  H  T  S  R  X
 V  Q  <  >  ?   Z  B  F  G  J
 
 y  p  o  u  :   k  d  l  c  w
 i  n  e  a  ,   m  h  t  s  r  x
 v  q  "  '  .   z  b  f  g  j

**Comment by michaeldickens** [comment]
Hi Ben, I'm not sure what you changed so I can't say. You can change the way layouts get printed by modifying this function: https://github.com/michaeldickens/Typing/blob/master/keyboard.c
